_resources/test-resource.md

```diff
@@ -10,7 +10,7 @@ topics:
 - Nuclear Escalation
 - Deterrence
 - Clandestine Capabilities
-optional: Optional Sample
+optional:
```

Comment:
The "optional" front matter here should really be called `url`. There was a mistake made in the Forestry templates. So it needs to be updated in the Forestry config, here, and where it's used in the includes.